drawingutility.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <QMap>
+
+namespace jbrush {
+
+// Enum class contains shape types that can be selected and created by the administrator
+enum class SelectableShapeType
+{
+    LINE,
+    RECTANGLE,
+    ELLIPSE,
+    TEXT
+};
+
+namespace jconstants {
+// Maps a selectable shape type with a descriptive string
+const QMap<jbrush::SelectableShapeType, QString> SELECTABLE_SHAPE_TYPE_MAP = {
+    {jbrush::SelectableShapeType::LINE, "Line"}, {jbrush::SelectableShapeType::RECTANGLE, "Rectangle"},
+    {jbrush::SelectableShapeType::ELLIPSE, "Ellipse"}, {jbrush::SelectableShapeType::TEXT, "Text"}
+};
+}
+
+// Class composed by the MainWindow with methods and fields that
+// enable the administrator edit the canvas rendering area by
+// adding shapes, removing shapes, and moving shapes
+class DrawingUtility
```

Comment:
Maybe a better name would be ``ShapeMenu``
In addition, you can probably make it inherit a ``QWidget`` so that you can create a ``.ui`` file for the menu.

Reply:
Not a bad idea.  I was thinking of making a class that Brendon could add to, but if you think it should be more self-contained to this branch, I'm down to try it out